apps/server/src/modules/authorization/authorization.service.spec.ts

```diff
@@ -191,27 +191,6 @@ describe('AuthorizationService', () => {
 	});
 
 	describe('hasPermissionByReferences', () => {
-		describe('when loader throws an error', () => {
```

Comment:
I guess this test should stay if you still want to throw the ForbiddenException.